wiki/Client/Program_files/de.md

```diff
@@ -0,0 +1,110 @@
+# osu!-Programmdateien
+
+*Siehe auch: [osu!-Dateiformate](/wiki/Client/File_formats)*
+
+![Die Dateistruktur des Installationsordners von osu! auf Windows und macOS](img/file_structure.jpg "Die Dateistruktur des Installationsordners von osu! auf Windows und macOS")
+
+Die **osu!-Programmdateien** sind eine Reihe von Dateien, die für die Ausführung von osu! erforderlich sind und die verschiedenen Aktivitäten des Benutzers speichern.
```

Comment:
I didn't want to use `verfolgen/aufzeichnen` in the traditional sense of `track`, because it makes it seem like osu! is some data kraken company, which is not true at all.